tests/openssl_moretest/host/CSchema_checker.cpp

```diff
@@ -0,0 +1,358 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <new>
+
+#include "CSchema_checker.h"
+#include "include_openssl.h"
+
+CSchemaChecker::CSchemaChecker(t_openssl_schema* schema, uint schema_size)
```

Comment:
Can you comment on why the `CSchemaChecker` is needed? I can infer what it's being used for in the current implementation of the tests, but nowhere in this PR is it made clear exactly what is being tested, why it is interesting, and why there is a desire to schematize OpenSSL functional signatures in this way.

It arguably gives you a "standard" way of allocating/deallocating parameters, but the tradeoff is that it makes each of the individual tests much more opaque in terms of understanding what parameters are actually being tested. Given that each function signature only seems to be associated with a single test, it's not obvious why the maintenance and readability overhead of this model is useful.